src/Playwright.Tests/Playwright.Tests.csproj

```diff
@@ -8,6 +8,7 @@
         <AssemblyName>Microsoft.Playwright.Tests</AssemblyName>
         <RootNamespace>Microsoft.Playwright.Tests</RootNamespace>
         <GenerateProgramFile>false</GenerateProgramFile>
+        <Nullable>enable</Nullable>
```

Comment:
If we enable this on the main project too, are there too many errors? That way we could eliminate the pragma directives in each file...

Reply:
Hundreds. Those are all warnings btw.